src/components/PopupWithForm.js

```diff
@@ -29,7 +29,8 @@ export default class PopupWithForm extends Popup {
     return inputsObject;
   }
 
```

Comment:
`    this._popupElement.removeEventListener("submit", this._handleFormSubmit);
`

In line 18

You don't add `this._handleFormSubmit` as the listener. You add `this._submitForm` in line 40, so you need to remove the same `this._submitForm`


```js
    this._popupElement.removeEventListener("submit", this._submitForm);


```